### DISCUSSION TITLE
Problems with docker

### DISCUSSION THREAD

**Discussion by guyindepurker** [QuorumDMS/ftp-srv#278] [Q&A]
Hi i have problem with docker container.
I create a image from project base ftp-srv,
try to connect to local host: 127.0.0.1 with the port that i want but i cant connect(try to connect to the project by ftp client program like filezilla)
maybe I dont do the right configuration for the project.
You have example project with docker or with the right config?

**Comment by matt-forster** [comment]
FTP within a containerized environment is actually a little harder than it seems on the surface - you need to control the ports available to the container based on your connection scheme, and make sure to allow both incoming and outgoing connections to the correct ports. You should be able to control these with the server settings and using the https://github.com/autovance/ftp-srv#pasv_min and https://github.com/autovance/ftp-srv#pasv_max options.

I cannot help you without seeing how your image is built, and the commands you are using to run the container - making sure the correct ports are exposed and you are linking them to your local network correctly.

I am transferring this to a discussion because at this point we don't officially support docker - but I am certainly willing to work through the problem if you can provide adequate context.
